storage-resize-images/extension.yaml

```diff
@@ -126,7 +126,7 @@ params:
     example: my-project-12345.appspot.com
     validationRegex: ^([0-9a-z_.-]*)$
     validationErrorMessage: Invalid storage bucket
-    default: ${param:STORAGE_BUCKET}
+    default: ${STORAGE_BUCKET}
```

Comment:
Is the new param syntax causing problems when generating release notes? Because #331 states that it should be the preferred syntax.

Reply:
Hi @rosariopfernandes! It actually caused a problem in the install flow:

![7nQZCVSykEmWFjM](https://user-images.githubusercontent.com/3759507/98271292-a61ba980-1f44-11eb-85b4-206064826cc9.png)

I'm not sure if that's a console bug, or a backend bug, so I'm just reverting back to `${STORAGE_BUCKET}` and will follow up with the rest of the extensions team to fix this going forward.

Reply:
@jhuleatt Ah, I see. Thanks for clarifying!